customersupport/templates/new-ticket-modal.html

```diff
@@ -0,0 +1,63 @@
+<div class="modal fade" id="newTicketModal" tabindex="-1" role="dialog" aria-labelledby="newTicketModalLabel">
+    <div class="modal-dialog" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span
+                        aria-hidden="true">&times;</span></button>
+                <h4 class="modal-title" id="newTicketModalLabel">New Ticket</h4>
+            </div>
+            <div class="modal-body">
+                <form>
+                    <div class="row">
+                        <div class="col-sm-12">
+                            <div class="form-group">
+                                <label for="titleForm">Ticket Name</label>
+                                <input type="text" class="form-control" id="titleForm" value="">
+                            </div>
+                        </div>
+                    </div>
+
+                    <div class="row">
+                        <div class="col-sm-12">
+                            <div class="form-group">
+                                <label for="status">Status</label>
+                                <select class="form-control" id="status">
+                                    <option>New</option>
+                                    <option>Pending</option>
+                                    <option>Closed - Successful</option>
+                                    <option>Closed - Failure</option>
+                                </select>
+                            </div>
+                        </div>
+                    </div>
+
+                    <div class="row">
+                        <div class="col-sm-12">
+                            <div class="form-group">
+                                <div id="tableDiv">
+                                    <table id="newTicketTable" class="table table-striped table-bordered">
+                                        <thead>
+                                        <tr>
+                                            <th>Serial Number</th>
+                                            <th>Phone Model</th>
+                                            <th>Refund</th>
+                                            <th>Replace</th>
+                                        </tr>
+                                        </thead>
+                                    </table>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                </form>
+            </div>
+            <div class="modal-footer">
+                <button type="button" class="btn btn-default" id="closeNewTicket" data-dismiss="modal">Close</button>
+                <button type="button" class="btn btn-primary" id="saveTicketChanges">Save changes</button>
+            </div>
+        </div>
+
+    </div>
+</div>
+</div>
```

Comment:
Remember in class when I asked why the page looked so weird on your screen?
Remove two of these `</div>` tags from the end and it will fix it.